releng/platform-definitions/platform-definition-2020-09/.project

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>platform-definition-2020-09</name>
+	<name>platform-definition-2020-12</name>

```

Comment:
I think this needs to be 09 again with another .project file added to the 2020-12 folder.

Reply:
Oops. Thanks!